airgun/views/contentview.py

```diff
@@ -60,3 +62,12 @@ class Details(SatTab):
         description = EditableEntry(name='Description')
         composite = ReadOnlyEntry(name='Composite?')
         force_puppet = EditableEntryCheckbox(name='Force Puppet')
+
+    @View.nested
+    class yumrepo(SatTabWithDropdown):
+        TAB_NAME = 'Yum Content'
+        SUB_ITEM = 'Repositories'
+
+        @View.nested
+        class repos(AddRemoveResourcesView):
```

Comment:
tip: turns out there's nicer way to do add nested view when no modifications needed:
```python
repos = View.nested(AddRemoveResourcesView)
```

Reply:
thanks, will definitely try